ui/package.json

```diff
@@ -3,45 +3,43 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@caraml-dev/ui-lib": "^1.7.6",
-    "@elastic/datemath": "5.0.3",
-    "@elastic/eui": "66.0.0",
-    "@emotion/react": "^11.9.0",
+    "@caraml-dev/ui-lib": "^1.13.0-build.1-d1f1876",
+    "@elastic/datemath": "^5.0.3",
+    "@elastic/eui": "^95.2.0",
+    "@emotion/css": "^11.11.2",
+    "@emotion/react": "^11.11.4",
     "@overgear/yup-ast": "^1.0.3",
-    "@sentry/browser": "5.15.5",
-    "js-yaml": "^4.0.0",
-    "mitt": "^2.1.0",
-    "moment": "^2.29.4",
+    "@sentry/browser": "^7.118.0",
```

Comment:
For some reason I can't upgrade this to v8 and above unlike what was done for MLP and Merlin (doesn't use craco) because craco/module federation somehow seems to have issues [resolving](https://github.com/caraml-dev/turing/actions/runs/9791175245/job/27034367025#step:8:24) this dependency (it only throws a warning but an error though):
```
No version specified and unable to automatically determine one. No version in description file (usually package.json). Add version to description file /home/runner/work/turing/turing/ui/node_modules/@sentry/browser/build/npm/esm/package.json, or manually specify version in shared config.
```

I've tried specifying the newer version explicitly in the shared config as described here but nothing worked:
- https://stackoverflow.com/questions/71197822/module-federation-react-and-apollo-3-warnings/78162319#78162319
- https://stackoverflow.com/questions/77068395/shared-module-warning-when-upgrading-swr-from-version-1-to-version-2-in-a-nextjs

Apparently there are other users facing this and the response given by the module federation people was that this needs to be fixed by the package managers (sentry in this case) themselves because it's a file in their package that doesn't have a version specified:
- https://github.com/module-federation/module-federation-examples/issues/1248
- https://github.com/webpack/webpack/issues/15864

I've just updated this package to the latest version I possibly can without the warning being thrown and breaking the CICD pipeline.

Reply:
same if we can add some of the breakage in comment

Reply:
Added one!